jena-arq/src/main/java/org/apache/jena/riot/out/NodeFormatterTTL_MultiLine.java

```diff
@@ -18,95 +18,120 @@
+        };
+        if ( !writeAsMultiLine(w, lex) ) {
+            writeLexicalSingleLine(w, lex, runnable);
+            return;
         }
-        writeLexicalMultiLine(w, lex, ()->{ w.print('@') ; w.print(langTag) ; }) ;
+        writeLexicalMultiLine(w, lex, runnable);
     }
 
     @Override
-    protected void writeLiteralLongForm(AWriter w, String lex, String datatypeURI)
-    {
-        if ( ! writeAsMultiLine(w, lex) ) {
+    protected void writeLiteralLongForm(AWriter w, String lex, String datatypeURI) {
+        Runnable runnable = () -> {
+            w.print("^^");
+            formatURI(w, datatypeURI);
+        };
+        if ( !writeAsMultiLine(w, lex) ) {
+            writeLexicalSingleLine(w, lex, runnable);
             super.writeLiteralOneLine(w, lex, datatypeURI);
-            return ;
+            return;
         }
-        writeLexicalMultiLine(w, lex, ()-> { w.print("^^") ; formatURI(w, datatypeURI) ; }) ;
+        writeLexicalMultiLine(w, lex, runnable);
     }
 
     private QuotedStringOutput chooseEscapeProcessor(String str) {
-        QuotedStringOutput proc = escapeProc2 ;
+        QuotedStringOutput proc = escapeProc2;
         if ( str.indexOf(CH_QUOTE2) >= 0 && str.indexOf(CH_QUOTE1) < 0 )
-            // Contains " but not ' so print using '-quotes.   
-            proc = escapeProc1 ;
-        return proc ;
+            // Contains " but not ' so print using '-quotes.
+            proc = escapeProc1;
+        return proc;
     }
 
-    /** Output a string and run the Runnable at the same indentation level */ 
+    /** Output a string and run the Runnable at the same indentation level */
     private void writeLexicalSingleLine(AWriter writer, String str, Runnable action) {
-        QuotedStringOutput proc = chooseEscapeProcessor(str) ;
-        proc.writeStr(writer, str); 
+        QuotedStringOutput proc = chooseEscapeProcessor(str);
+        proc.writeStr(writer, str);
+        if ( action != null )
+            action.run();
     }
 
-    /** Output a string and run the Runnable at the same indentation level */ 
+    /** Output a string and run the Runnable at the same indentation level */
     private void writeLexicalMultiLine(AWriter writer, String str, Runnable action) {
-        QuotedStringOutput escapeProc = chooseEscapeProcessor(str) ;
-        int indent = -1 ; 
-        IndentedWriter iw = null ;
+        QuotedStringOutput escapeProc = chooseEscapeProcessor(str);
+        int indent = -1;
+        IndentedWriter iw = null;
         if ( writer instanceof IndentedWriter ) {
-            iw = (IndentedWriter)writer ;
-            iw.pad(); 
-            indent = iw.getAbsoluteIndent() ;
+            iw = (IndentedWriter)writer;
+            iw.pad();
+            indent = iw.getAbsoluteIndent();
             iw.setAbsoluteIndent(0);
         }
-        escapeProc.writeStrMultiLine(writer, str) ;
+        escapeProc.writeStrMultiLine(writer, str);
 
         if ( action != null )
```

Comment:
This `null` check is unnecessary, as all callers of this `private` method supply a `Runnable`. And that is sorta the point of the above-defined `noop` `Runnable` - it obviates the need for a `null` check. : )